recipes/kernax/meta.yaml

```diff
@@ -0,0 +1,53 @@
+package:
```

Comment:
You need to set the version as a jinja variable. See:
https://github.com/conda-forge/staged-recipes/blob/main/recipes/example/meta.yaml#L9

I would then use the variable `{{ version }}` everywhere that it appears (i.e. in the `url` under `source` and `version` under `package`).

Reply:
Thanks for helping me out. It should be corrected.